src/commands/docs/generateDocs.ts

```diff
@@ -59,10 +65,27 @@ export async function generateDocs(targetName: string, outDirectory: string) {
     )
   }
 
+  let PROJECT_NAME = 'SAS JS'
+  let PROJECT_BRIEF = 'Command line interface for SASjs'
+  let PROJECT_REPO = 'https://github.com/sasjs/cli'
+
+  const packageJsonContent = await readFile(
+    path.join(process.projectDir, 'package.json')
+  )
+  if (packageJsonContent) {
+    const packageJson = JSON.parse(packageJsonContent)
+    PROJECT_NAME = packageJson.name
+    PROJECT_BRIEF = packageJson.description
+    PROJECT_REPO = packageJson.homepage
```

Comment:
perhaps we need a default in case of missing or empty?  such as "Please add 'homepage' to your package.json"

Reply:
https://github.com/sasjs/cli/pull/481/files/4377c1b724b411bf5ab9e831fc76ed821af5b35c#diff-e87d4a4f7828949b17fee99acd797cc46fa8cfd7d5ff48c451972cbb5c22a0feR68

Are we good with these as default values?